src/components/Conversation/ContentCard.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+import { useState } from "react";
+import { ViewPoint } from "@/types/conversations.types";
+import {
+    HandThumbUpIcon,
+    HandThumbDownIcon,
+    ArrowUpCircleIcon,
+} from "@heroicons/react/24/solid";
+type ContentCardProps = {
+    viewpoint: ViewPoint;
+};
+export default function ContentCard({ viewpoint }: ContentCardProps) {
+    const [thumbsup, setThumbsup] = useState(false);
+    const [thumbsdown, setThumbsdown] = useState(false);
+    const [arrowup, setArrowup] = useState(false);
+    const [thumbsupNum, setThumbsupNum] = useState(viewpoint.thumbsup);
+    const [thumbsdownNum, setThumbsdownNum] = useState(viewpoint.thumbsdown);
+    const [arrowupNum, setArrowupNum] = useState(viewpoint.up);
+
+    function HandleThumbsup() {
+        if (thumbsdown === true) {
+            setThumbsdown(false);
+            setThumbsup(true);
+            setThumbsupNum(thumbsupNum + 1);
+            setThumbsdownNum(thumbsdownNum - 1);
+            return;
+        } else if (thumbsup === true) {
+            setThumbsup(false);
+            setThumbsupNum(thumbsupNum - 1);
+            return;
+        } else if (thumbsup === false) {
+            setThumbsup(true);
+            setThumbsupNum(thumbsupNum + 1);
+            return;
+        }
+    }
+    function HandleThumbsdown() {
+        if (thumbsup === true) {
+            setThumbsup(false);
+            setThumbsdown(true);
+            setThumbsdownNum(thumbsdownNum + 1);
+            setThumbsupNum(thumbsupNum - 1);
+            return;
+        } else if (thumbsdown === true) {
+            setThumbsdown(false);
+            setThumbsdownNum(thumbsdownNum - 1);
+            return;
+        } else if (thumbsdown === false) {
+            setThumbsdown(true);
+            setThumbsdownNum(thumbsdownNum + 1);
+            return;
+        }
+    }
+    function HandleArrowup() {
+        if (arrowup === true) {
+            setArrowup(false);
+            setArrowupNum(arrowupNum - 1);
+            return;
+        } else {
+            setArrowup(true);
+            setArrowupNum(arrowupNum + 1);
+            return;
+        }
+    }
+
+    console.log(thumbsup);
+
+    return (
+        <>
```

Comment:
Consider returning a div instead of a fragment since it makes sense that reactions are put in their own div